<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nauczyciel</class>
 <widget class="QMainWindow" name="Nauczyciel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(33, 33, 33, 255), stop:1 rgba(48, 48, 48, 255));
color: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="TeacherWindow">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QFrame" name="TeachSideBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>720</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>160</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="TeachVert">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="TeachGrdBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Oceny</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachSchdBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Plan Zajęć</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachAttndcBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Frekwencja</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachNoteBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Uwagi</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="TeachLogoutBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Wyloguj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="TeachGrdMenu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>-1</y>
      <width>1121</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TeachHori1">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QComboBox" name="TeachClassCBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="TeachSubjCBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="TeachClassLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>78</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Klasa</string>
     </property>
    </widget>
    <widget class="QLabel" name="TeachSubjLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>78</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Przedmiot</string>
     </property>
    </widget>
    <widget class="QTableView" name="TeachGrdList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>1101</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1101</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TeachHori">
      <item>
       <widget class="QPushButton" name="TeachAddGrdBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Dodaj ocenę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachDelGrdBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Usuń ocenę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachEditGrdBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Edytuj ocenę</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="TeachAttndcMenu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>1121</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>911</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TeachHori3">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QComboBox" name="TeachClassCBox_4"/>
      </item>
      <item>
       <widget class="QComboBox" name="TeachSubjCBox_4"/>
      </item>
      <item>
       <widget class="QComboBox" name="TeachDateCBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="TeachClassLabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Klasa</string>
     </property>
    </widget>
    <widget class="QLabel" name="TeachSubjLabel_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Przedmiot</string>
     </property>
    </widget>
    <widget class="QTableView" name="TeachAttndcList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>1101</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="TeachSubjLabel_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>1101</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="TeachAddAttndcBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Dodaj obecność</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachDelAttndcBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Usuń obecność</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachEditAttndcBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Zmień obecność</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="TeachNoteMenu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>1121</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TeachHori3_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QComboBox" name="TeachClassCBox_5"/>
      </item>
      <item>
       <widget class="QComboBox" name="TeachStudCBox_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="TeachClassLabel_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Klasa</string>
     </property>
    </widget>
    <widget class="QLabel" name="TeachSubjLabel_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Uczeń</string>
     </property>
    </widget>
    <widget class="QTableView" name="TeachNotesList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>1101</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>1101</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="TeachAddNoteBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Dodaj uwagę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachDelNoteBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Usuń uwagę</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TeachEditNoteBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Zmień uwagę</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="TeachSchdMenu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>1121</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QScrollArea" name="TeachScroll">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>99</y>
       <width>1101</width>
       <height>571</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="TeachScrollContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1099</width>
        <height>569</height>
       </rect>
      </property>
      <widget class="QWidget" name="TeachGridWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1101</width>
         <height>571</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1101</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout"/>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>TeachSideBar</zorder>
   <zorder>TeachAttndcMenu</zorder>
   <zorder>TeachGrdMenu</zorder>
   <zorder>TeachNoteMenu</zorder>
   <zorder>TeachSchdMenu</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
